pkg/ui/console/client.go

```diff
@@ -400,9 +400,10 @@ func (c Client) RunPolicies(ctx context.Context, policySource, outputDir string,
 		DBPersistence: dbPersistence,
 	})
 	if resp == nil {
-		analytics.Capture("policy run", c.Providers, policiesToRun, diags)
-	} else {
-		analytics.Capture("policy run", c.Providers, resp.Policies, diags)
+		policiesToRun = resp.Policies
```

Comment:
[blocking] I believe `resp` is `nil` here

Reply:
good catch meant `!=`